yt/data_objects/tests/test_center_squeeze.py

```diff
@@ -0,0 +1,57 @@
+from yt.testing import assert_array_equal, fake_amr_ds, fake_particle_ds, fake_random_ds
+
+
+def test_center_squeeze():
+    # tests that selected values match when supplying center arrays of different shapes
+    # to the data container.
+
+    # list of fields to populate fake datasets with
+    fldz = ("density", "velocity_x", "velocity_y", "velocity_z")
+
+    # create and test amr, random and particle data
+    check_single_ds(fake_amr_ds(fields=fldz))
+    check_single_ds(fake_random_ds(16, fields=fldz))
+    check_single_ds(fake_particle_ds(npart=100), check_morton=False)
+
+
+def check_single_ds(ds, check_morton=True):
```

Comment:
I'd suggest removing the default value here and explicitly set the parameter to True where required.